client/wc-api/items/utils.js

```diff
@@ -20,9 +20,13 @@ export function searchItemsByString( select, endpoint, search ) {
 	let isError = false;
 	search.forEach( searchWord => {
 		const query = {
-			search: searchWord,
 			per_page: 10,
 		};
+		if ( 'products' === endpoint ) {
+			query.product_search = searchWord;
```

Comment:
Do we have to rename `search` to `product_search` here? Since we're already overriding `get_collection_params()`, can't we handle the `search` parameter there?

Reply:
@jeffstieler Will do!